models/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var fs        = require('fs');
+var path      = require('path');
+var Sequelize = require('sequelize');
+var basename  = path.basename(__filename);
+var env       = process.env.NODE_ENV || 'development';
+var config    = require(__dirname + '/../config/config.json')[env];
+var db        = {};
+
+if (config.use_env_variable) {
+  var sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  var sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs
+  .readdirSync(__dirname)
+  .filter(file => {
+    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
```

Comment:
Think these changes were made in #21

Reply:
ok.  Looks like i created a branch off of another branch.  Ill make a new pr and tag you.

Reply:
ah gotcha. branching off a working branch can really fuck up your git merge if you commit from both branches and try to merge. It will remove code and not transfer all changes/override commits so careful with it. Always branch off of the latest master